tv_apps/smart-system/js/value_selector/value_selector.js

```diff
@@ -529,12 +531,8 @@
       });
     }
 
-    var separator = ':';
-    var minutesPosition = localeTimeFormat.indexOf('%M');
-    if (minutesPosition > 0) {
-      separator = localeTimeFormat.substr(minutesPosition - 1, 1);
-    }
-    this.elements.hoursMinutesSeparator.textContent = separator;
+    this.elements.hoursMinutesSeparator.setAttribute('data-l10n-id',
+      'hourMinutesSeparator');
```

Comment:
It's been added to `shared/locales/date/date.properties` - same file that l10n_date.js was using, so you should already have this file tracked.